pkg/token/execCredentialPlugin.go

```diff
@@ -28,6 +32,16 @@ type execCredentialPlugin struct {
 }
 
 func New(o *Options) (ExecCredentialPlugin, error) {
+	env := os.Getenv(execInfoEnv)
+	fmt.Fprintln(os.Stderr, os.Getenv(execInfoEnv))
+	var execCredential clientauthentication.ExecCredential
+	error := json.Unmarshal([]byte(env), &execCredential)
+	if error != nil {
+		return nil, fmt.Errorf("cannot convert to ExecCredential: %w", error)
+	}
+	if !execCredential.Spec.Interactive && o.LoginMethod == "DeviceCodeLogin" {
```

Comment:
are you sure it's `"DeviceCodeLogin"` instead of the const?

Reply:
I checked it and you are right. Changed to  DeviceCodeLogin const. Thanks!